packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-task-base-types.ts

```diff
@@ -151,16 +182,21 @@ export interface OutputDataConfig {
 }
 
 /**
+ * Specifies a limit to how long a model training job can run.
+ * When the job reaches the time limit, Amazon SageMaker ends the training job.
+ *
  * @experimental
  */
 export interface StoppingCondition {
     /**
      * The maximum length of time, in seconds, that the training or compilation job can run.
      */
-    readonly maxRuntime?: Duration;
+    readonly maxRuntime: Duration;
```

Comment:
By any chance, is this a breaking change? Should we announce it as a breaking change?

Reply:
this was unintended... thought I'd removed it... good catch.
default is an hour, will update